radeon-profile/ioctl_amdgpu.cpp

```diff
@@ -77,8 +77,7 @@ bool amdgpuIoctlHandler::getCoreClock(int *data) const {
 #ifdef AMDGPU_INFO_SENSOR_GFX_SCLK
     return getSensorValue(data, sizeof(data), AMDGPU_INFO_SENSOR_GFX_SCLK);
 #else
-    Q_UNUSED(data);
-    return false;
+    return getClocks(data, NULL);
```

Comment:
Ok, I will revert this. 
But this opens another problem: getClocks() is also supposed to return core and memory clocks, so the AMDGPU_INFO_VCE_CLOCK_TABLE should be completely deleted (and at this point getClocks() would become useless, so we might even delete it from ioctlHandler).